packages/server/utils/azureDevOps/azureDevOpsFieldTypeToId.ts

```diff
@@ -1,8 +1,34 @@
+const effortField = '/fields/Microsoft.VSTS.Scheduling.Effort'
+const remainingWorkField = '/fields/Microsoft.VSTS.Scheduling.RemainingWork'
+const storyPointsField = '/fields/Microsoft.VSTS.Scheduling.StoryPoints'
+const originalEstimateField = '/fields/Microsoft.VSTS.Scheduling.OriginalEstimate'
+const sizeField = '/fields/Microsoft.VSTS.Scheduling.Size'
+
 // Agile fields in map, TODO add Scrum and CMMI fields
```

Comment:
+1 Looks like Scrum and CMMI fields are now present? Can this comment be removed?

Reply:
Done